btc-transaction-helper.js

```diff
@@ -295,6 +295,16 @@ class BtcTransactionHelper {
         return await this.nodeClient.getBlockCount();
     }
 
+    /**
+     * 
+     * @param {string} blockHash 
+     * @param {boolean} jsonEncoded defaults to true. If false, the block header will be returned as a hex string.
+     * @returns {BlockHeader | string} blockHeader
+     */
+    getBlockHeader(blockHash, jsonEncoded = false) {
```

Comment:
Why not set the `jsonEncoded` param as `true` by default? It is more useful that way.